app/lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -97,7 +97,7 @@ def initialize(options = {})
 
   class HttpChecker < Checker
     def call
-      if uri.host.nil?
+      if uri.host.blank?
```

Comment:
Should there be corresponding tests for this change?

Reply:
There kind of already is in the sense that the tests start failing after https://github.com/alphagov/link-checker-api/pull/436/commits/5e095ca0bfd9aae18739ae923a83baa6eebd0462 but work again after https://github.com/alphagov/link-checker-api/pull/436/commits/bd4ca57ca6b0a74b5ad83d09f157f66344fe8221. Do you think that's enough?

Reply:
Ah, I missed that context. Yes that seems fine! 👍